### PR TITLE
#170972171 Signup using database

### PR DESCRIPTION
**Pull request by Muhire-Josue**
#### What does this PR do?

- User can sign up in the database

#### Description of Task to be completed?

- It uses postgres database to store user information

#### How should this be manually tested?

- Use this link in postman: https://anounce-it.herokuapp.com/api/v1/auth/signup

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [170972171](https://www.pivotaltracker.com/story/show/170972171)

#### Screenshots (if appropriate)
signup
user-duplication
welcome

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+        [
+            "@babel/preset-env",
+            {
+                "targets": {
+                    "node": "current"
+                }
+            }
+        ]
+    ]
 }
```
Comment (j4l13n) [inline .babelrc]: Fix this new line issue

File: server/Tests/userTEST.test.js [hunk 2]
```diff
@@ -45,48 +45,47 @@ describe('User tests', () => {
...
-      .post('/api/v1/auth/signup')
+      .post('/api/v2/auth/signup')
       .send(user)
       .end((error, res) => {
         res.body.status.should.be.equal(409);
         expect(res.body.error).to.equal('Email already exists');
         done();
       });
   });
+  // it('Should login a user', (done) => {
```
Comment (j4l13n) [inline server/Tests/userTEST.test.js]: Remove the commented code before you proceed

**Review by j4l13n — COMMENTED** [review]
Good job @Muhire-Josue, Fix all small issues that you have before you merge this PR

File: server/Routes/userRoutes.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import userFound from '../Middlewares/userFound';
 import incorrectPassword from '../Middlewares/incorrectPassword';
 
 const router = express.Router();
-router.get('/api/v1/', userController.welcomeToAPI);
-router.post('/api/v1/auth/signup', userValidation, userDuplication, userController.signUp);
+router.get('/api/v2/', userController.welcomeToAPI);
+router.post('/api/v2/auth/signup', userValidation, userDuplication, userController.signUp);
 router.post('/api/v1/auth/signin', userFound, incorrectPassword, userController.login);
 export default router;
```
Comment (j4l13n) [inline server/Routes/userRoutes.js]: Fix this as well

**Review by j4l13n — COMMENTED** [review]
Fix line issues

File: server/Routes/userRoutes.js (hunk 3 above)
Comment (Muhire-Josue) [reply server/Routes/userRoutes.js]: Should I leave it to v1 instead of v2?

**Pull request merged**